cmd/doctor.go

```diff
@@ -124,13 +125,18 @@ func runRecreateTable(ctx *cli.Context) error {
 }
 
 func runDoctor(ctx *cli.Context) error {
+	stdCtx, cancel := installSignals()
+	defer cancel()
```

Comment:
What's the difference to the previous approach were the signals were installed after the loggers were deleted?

Reply:
You can read the code. 

I do not see difference. I do not like to put non-related code after the logger deletion.